.travis.yml

```diff
@@ -0,0 +1,34 @@
+sudo: required
+
+language: python
+
+services:
+  - docker
+
+env:
+  global:
+    - PORTS="8069:8069"
+    - LINKS="db:db"
+    - DAEMONIZE="1"
+    - ODOO_VERSION="10.0"
```

Comment:
[Example of pushing multiple tags](https://github.com/LasLabs/docker-odoo/blob/master/.travis.yml#L21)